pages/task/content/status.js

```diff
@@ -77,46 +82,54 @@ module.exports = {
     action: 'Recommend for rejection',
     hint: {
       application: 'The application will be sent to a Licensing Officer who will action your recommendation.',
-      amendment: 'The amendment will be sent to a Licensing Officer who will action your recommendation.'
+      amendment: 'The amendment will be sent to a Licensing Officer who will action your recommendation.',
+      revocation: 'The revocation will be sent to a Licensing Officer who will action your recommendation.'
     },
     log: 'Recommended by'
   },
   resubmitted: {
     state: 'Submitted',
     action: {
       application: 'Submit application',
-      amendment: 'Submit amendment'
+      amendment: 'Submit amendment',
+      revocation: 'Submit revocation'
     },
     hint: {
       application: 'Your application will be sent to the Home Office for review.',
-      amendment: 'Your amendment will be sent to the Home Office for review.'
+      amendment: 'Your amendment will be sent to the Home Office for review.',
+      revocation: 'Your revocation will be sent to the Home Office for review.'
     },
     log: 'Resubmitted by'
   },
   resolved: {
     state: 'Approved',
     action: {
       application: 'Grant licence',
-      amendment: 'Amend licence'
+      amendment: 'Amend licence',
+      revocation: 'Revoke licence'
     },
     hint: {
       application: 'A new licence will be granted.',
-      amendment: 'The existing licence will be updated.'
+      amendment: 'The existing licence will be updated.',
+      revocation: 'The existing licence will be revoked.'
     },
     log: {
       application: 'Granted by',
-      amendment: 'Approved by'
+      amendment: 'Approved by',
+      revocation: 'Revoked by'
     }
   },
   rejected: {
     state: 'Rejected',
     action: {
       application: 'Reject application',
-      amendment: 'Reject amendment'
+      amendment: 'Reject amendment',
+      revocation: 'Reject revocation'
     },
     hint: {
       application: 'The applicant will need to create a new application to apply for this type of licence in the future.',
-      amendment: 'The applicant will need to create a new amendment to submit these changes again.'
+      amendment: 'The applicant will need to create a new amendment to submit these changes again.',
+      revocation: 'The applicant will need to create a new revocation for this licence to be revoked.'
```

Comment:
There's potential for deduping a lot of the content in this file, but meh...

Reply:
this is by choice - the simpler the file is, the more Phil can own it